### DISCUSSION TITLE
Azure Devops Agents upgraded to 3.14 broke my 3.11 - can now repro locally after upgrading

### DISCUSSION THREAD

**Discussion by dombat** [wixtoolset/issues#8038] [Questions] [answered]
I've made no changes to my solution, but found that my Azure Devops pipeline was breaking. To cut a long story short, the Azure DevOps agent was upgraded to 3.14 ( see https://github.com/actions/runner-images/issues/9426 )

After upgrading to 3.14 locally, I now get the same error.

1st Error:
Location of symbol related to previous error.	NetFx472.wxs	line 21	 (I dont have a file NetFx472.wxs locally).

2nd error:
Duplicate symbol 'Property:WIX_IS_NETFRAMEWORK_472_OR_LATER_INSTALLED' found. This typically means that an Id is duplicated. Check to make sure all your identifiers of a given type (File, Component, Feature) are unique.

Is there any documentation on this change? Do I need to do the WIX_IS_NETFRAMEWORK_472_OR_LATER_INSTALLED or is that now defined in NetFx472.wxs ?

**Comment by chrpai** [comment, marked as answer]
Prior to 3.14, 3.11 didn't have support for various .NET Framework versions so people defined it themselves. In 3.14 these properties are added to the NetFx extension so you need to remove your own definition or change it to a unique property name.

NetFx472.wxs is inside the extension itself.  Your property definition could be anywhere in your code.  Search for WIX_IS_NETFRAMEWORK_472_OR_LATER_INSTALLED across all of your wxs files.
